GETTING_STARTED.md

```diff
@@ -0,0 +1,385 @@
+      // ...
+    }
+    // ... more columns to come
+  ]
+}
+```
+
+### Retrieving rows
+
+`GET /tables/:tableid/rows`
+
+Call this endpoint to retrieve all rows of a specific table. Most important part of a row object is `id` and `values`. The `id` is the unique row ID which is needed to identify the row. The `values` array contains one value object for each column. The order of `values` is exactly the same as of `columns`. In this case the table has two columns and therefore each `values` array has the length two. The different [data types](#22-data-types--column-kinds) will be described later on.
+
+```
+{
+  "page": { // result can be paged with two query parameters offset and limit
+    "offset": null,
+    "limit": null,
+    "totalSize": 18 // total size of this table
+  },
+  "rows": [
+    { // row object
+      "id": 24, // Unique column ID
+      "values": [
+        {
+          "de": "Schweden",
+          "es": "Suecia",
+          "fr": "Suède",
+          "it": "Svezia"
+        },
+        "2nd value"
+      ]
+    },
+    {
+      "id": 25,
+      "values": [
+        {
+          "de": "Deutschland",
+          "en": "Germany"
+        },
+        "2nd value"
+      ]
+    }
+    // ... more rows to come
+  ]
+}
+```
+
+### Retrieving a specific column
+
+`GET /tables/:tableid/columns/:columnid`
+
+Call this endpoint to retrieve the definition of one specific column. Same column object definition as above. 
+
+```
+{
+  "id": 1,
+  "ordering": 1,
+  "name": "name",
+  "kind": "shorttext"
+  // ...
+}
+```
+
+### Retrieving a specific row
+
+`GET /tables/rows/:rowid`
+
+Call this endpoint to retrieve a specific row. Same row object definition as above.
+
+```
+{
+  "id": 24,
+  "values": [
+    {
+      "de": "Schweden",
+      "es": "Suecia",
+      "fr": "Suède",
+      "it": "Svezia"
+    },
+    "2nd value"
+  ]
+}
+```
+
+### Retrieving a specific cell
+
+`GET /tables/:tableid/columns/:columnid/rows/:rowid`
+
+Call this endpoint to retrieve a single value of a specific cell. Value object can be different for the various data types.
+
+```
+{
+  "value": {
+    "de": "Schweden",
+    "es": "Suecia",
+    "fr": "Suède",
+    "it": "Svezia"
+  }
+}
+```
+
+## 2.2. Data types & column kinds
+
+Each column has a specific data type defined in its `kind` field. There are a number of different data types which will be described in this chapter.
+
+### Primitive data types
+
+Currently there are a few primitive data types. All primitive data types can be used in a multi-language column or a language neutral column.
+
+* `text`, `shorttext`, and `richtext`
+* `numeric` and `currency`
+* `date` and `datetime`
+* `boolean`
+
+#### Examples 
+
+For text there are three different column kinds. `text`, `shorttext`, and `richtext`. All three are syntactically the same but semantically different. Frontend interfaces can use this information to display the text differently.
+
+* `shorttext` should only contain a word or a short sentence
```

Comment:
Specificallly, shorttext should not contain line breaks, text may contain anything but is not expected to be formatted, richtext -> md

Reply:
Ok, added it the bullet points. 👍